doc/copilot.txt

```diff
@@ -106,7 +106,7 @@ copilot#Accept().  Here's an example with CTRL-J:
 <
 Lua version:
 >
-        vim.keymap.set('i', '<C-J>', 'copilot#Accept("\<CR>")', {
+        vim.keymap.set('i', '<C-J>', 'copilot#Accept(\"<CR>\")', {
```

Comment:
Nitpick - We don't even need to escape the double quotes: 

`        vim.keymap.set('i', '<C-J>', 'copilot#Accept("<CR>")', {`